packages/react/src/portal.tsx

```diff
@@ -27,6 +27,12 @@ export const Portal: (props: PortalProperties & RefAttributes<ComponentInternals
         () => ({
           events: { compute: uvCompute.bind(null, imageRef), priority: eventPriority },
           size: { width: 1, height: 1, left: 0, top: 0 },
+          // We have our own camera in here, separate from the main scene.
+          camera: (() => {
```

Comment:
nit pick: I thinks its easier to read if you put the camera initialization at the start of the function
e.g. `() => { const camera = ...; return { ..., camera } }`